update/stock_db.js

```diff
@@ -0,0 +1,47 @@
+//---CONFIGURE AUTHENTICATION HERE--------------------
+var CLOUDANT_KEY      = 'XXXXXXXXX';
+var CLOUDANT_PASSWORD = 'YYYYYYYYY';
+var CLOUDANT_ACCESS   = 'someurl-bluemix.cloudant.com'
+var DB_NAME = 'DB_NAME';
+//----------------------------------------------------
+
+const Cloudant = require('cloudant');
+const utils = require('./utils.js');
+
+var cloudant = Cloudant({
+  account: CLOUDANT_ACCESS,
+  key: CLOUDANT_KEY,
+  password: CLOUDANT_PASSWORD
+});
+var db = cloudant.db.use(DB_NAME);
+
+function getDocs(callback) {
+  db.list({
+    //this field is needed to return all doc data
+    include_docs: true
+  }, function(err, data) {
+    if (utils.isFunc(callback)) {
+      var docs = data.rows.map(function(row) {
+        return row.doc;
+      });
+      callback(err, docs);
+    }
+  });
+}
+
+//include _id and _rev of existing doc in the doc to perform an update
```

Comment:
this comment doesn't seem to apply? where are `_id` and `_rev`? in the `insertOrUpdateDoc` function?

Reply:
By "in the doc" i meant as members of the doc param. I could reword for clarity